src/docs/contributing/approach/sdk-docs/write-performance.mdx

```diff
@@ -31,15 +31,21 @@ This file is `index.mdx`. It explains how to install or set up (if appropriate)
 
 To develop content for this file:
 
-1. Determine if the SDK needs an installation step or a property set to enable tracing. If so, add the SDK to `Supported` list in the `<PlatformSection>`, then add a code sample to `performance/enable-tracing/`. If not, add the SDK to `notSupported` list, which will prevent the **Enable Tracing** heading from displaying for this SDK.
+1. If the SDK is still a work-in-progress and is exposed as an experimental or an early access feature, add the platform to the `<PlatformSection>` wrapping the appropriate alert located at the beginning of the content for the page.
```

Comment:
Sorry about this, this is referring to content that is currently exclusive to https://github.com/getsentry/sentry-docs/pull/4640. I'll add this in to the PR so it's less confusing.

Reply:
I've added this experimental note in the doc being referenced so this point isn't talking about something that just isn't present.

I've also added had a sibling section in custom instrumentation that also warns about the possibly experimental/wip state of an SDK. It's possible that this might've been excessive; I added it assuming that it was possible that users were finding their way directly to the custom instrumentation page instead of starting in Set Up.